_posts/2015-06-12-topics.md

```diff
@@ -871,3 +878,238 @@ This topic is deprecated. Please use [shipment:packed](#shipment_packed) instead
     }
 }
 ```
+<h3 id="tracker_updated">
+    tracker:updated
+</h3>
+
+```json
+{
+    "topic" : "tracker:updated",
+    "message" : {
+        "order_ref" : "9099",
+        "order_unique_id" : "100000018",
+        "shipment_unique_id" : "100000007",
+        "warehouse_id": 1,
+        "package_id" : 99,
+        "source" : "shopify",
+        "carrier" : "fedex",
+        "service" : "FEDEX_2_DAY",
```

Comment:
Remove since this is more a shipment-level thing and may not be precisely related to the package/tracker level datain - for example when buying shipping from Amazon the service is determined at time of label creation and uses Amazon codes.